### DISCUSSION TITLE
Replace Freenode link with Liberachat

### DISCUSSION THREAD

**Discussion by LiveLM** [bromite/bromite#1164] [Ideas]
When you first launch Bromite, the Chrome homepage is populated with a couple of useful links: The project homepage, the EFF, Freenode and some others.
![bromite](https://user-images.githubusercontent.com/34987841/119301185-99953f00-bc51-11eb-8cea-2b037dd49f14.png)  

A few days ago, Freenode suffered what many consider to be a "Hostile takeover", causing many of the staffers to resign and move on to create their own alternative: Liberachat.
It might be a good idea to either replace the Freenode link with Liberachat, or to remove the link from Bromite entirely.  

If you want to read more on what happened:  
[The Freenode resignation FAQ](https://gist.github.com/joepie91/df80d8d36cd9d1bde46ba018af497409/)
[Freenode now belongs to Andrew Lee, and I'm leaving for a new network.](https://www.kline.sh/)
[Freenode IRC operators now engaging in routine abuses of power](https://www.devever.net/~hl/freenode_abuse)
[Wikipedia: Freenode Ownership change and conflict](https://en.wikipedia.org/wiki/Freenode#Ownership_change_and_conflict)
[Liberachat homepage](https://libera.chat/)

**Comment by csagan5** [comment]
I will consider this for next release.
